dbfit-java/oracle/src/main/java/dbfit/environment/OracleStoredProcedureCall.java

```diff
@@ -15,13 +15,13 @@
 
 import dbfit.util.Direction;
 
-public class OracleStoredProcedureCall extends DbStoredProcedureCall {
+public class OracleStoredProcedureCall extends DefaultDbStoredProcedureCall {
```

Comment:
Any ideas for renaming these at this point? May be drop the `Db` prefix off of the interface or the classes...?